docs/admin/runai-setup/cluster-setup/cluster-install.md

```diff
@@ -27,6 +26,34 @@ On the next page:
 * (SaaS and remote self-hosted cluster only) Install a trusted certificate to the domain entered above.
 *  Run the [Helm](https://helm.sh/docs/intro/install/) command provided in the wizard.
 
+## Cluster Table
```

Comment:
@jasonnovichRunAI I think we should set the new section after "Validate the installation" and not before